strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/AbstractRepositoryProvider.java

```diff
@@ -215,7 +216,11 @@ public void commit(RepositoryStreamWriteContext ctx) throws IOException
         
         CountingOutputStream cos = StreamUtils.findSource(CountingOutputStream.class, ctx.getStream());
         artifactEntry.setSizeInBytes(cos.getByteCount());
-        
+
+        LayoutOutputStream los = StreamUtils.findSource(LayoutOutputStream.class, ctx.getStream());
+        artifactEntry.getChecksums().clear();
```

Comment:
Why is this invocation of `clear()` necessary? It seems redundant.

Reply:
@carlspring Frankly speaking I do not know. @sbespalov advised to add it.